index.html

```diff
@@ -0,0 +1,174 @@
+          <code class="graybackground">
+            x = 42.
+          </code><br>
+          This always declares a global variable. It generates a strict JavaScript warning. You shouldn't use this variant.<br>
+          With the keyword let. For example,<br>
+          <code class="graybackground">
+            let y = 13.
+          </code><br>
+          This syntax can be used to declare a block scope local variable. See Variable scope below.
+        </p>
+
+      </section>
+      <section id="Variable_scope" class="main-section">
+        <header ><h1>Variable scope</h1></header>
+        <p>
+          When you declare a variable outside of any function, it is called a global variable, because it is available to any other code in the current document. When you declare a variable within a function, it is called a local variable, because it is available only within that function.<br>
+          JavaScript before ECMAScript 2015 does not have block statement scope; rather, a variable declared within a block is local to the function (or global scope) that the block resides within. For example the following code will log 5, because the scope of x is the function (or global context) within which x is declared, not the block, which in this case is an if statement.<br>
+          <code class="graybackground">
+            if (true) { var x = 5; } console.log(x); // 5
+          </code><br>
+          This behavior changes, when using the let declaration introduced in ECMAScript 2015.<br>
+          <code class="graybackground">
+            if (true) { let y = 5; } console.log(y); // ReferenceError: y is not defined
+          </code>
+        </p>
+
+      </section>
+      <section id="Global_variables" class="main-section">
+        <header ><h1>Global variables</h1></header>
+        <p>
+          When you declare a variable outside of any function, it is called a global variable, because it is available to any other code in the current document. When you declare a variable within a function, it is called a local variable, because it is available only within that function.<br>
+          JavaScript before ECMAScript 2015 does not have block statement scope; rather, a variable declared within a block is local to the function (or global scope) that the block resides within. For example the following code will log 5, because the scope of x is the function (or global context) within which x is declared, not the block, which in this case is an if statement.<br>
+          <code class="graybackground">
+            if (true) { var x = 5; } console.log(x); // 5
+          </code>
+          This behavior changes, when using the let declaration introduced in ECMAScript 2015<br>
+          <code class="graybackground">
+            if (true) { let y = 5; } console.log(y); // ReferenceError: y is not defined
+          </div>
+        </p>
+
+      </section>
+    </div>
+
+    </main>
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Please remove the unnecessary white spaces. It improves code structure and readability. 😁